diff -up ispc-1.13.0/CMakeLists.txt.clang ispc-1.13.0/CMakeLists.txt
--- ispc-1.13.0/CMakeLists.txt.clang	2020-05-31 23:42:24.404485131 +0200
+++ ispc-1.13.0/CMakeLists.txt	2020-05-31 23:43:05.600482477 +0200
@@ -218,7 +218,7 @@ if (WASM_ENABLED)
     list(APPEND ISPC_TARGETS wasm-i32x4)
 endif()
 
-set(CLANG_LIBRARY_LIST clangFrontend clangDriver clangSerialization clangParse clangSema clangAnalysis clangAST clangBasic clangEdit clangLex)
+set(CLANG_LIBRARY_LIST clang-cpp)
 set(LLVM_COMPONENTS engine ipo bitreader bitwriter instrumentation linker option)
 
 if (${LLVM_VERSION_NUMBER} VERSION_GREATER_EQUAL "10.0.0")
