diff -up ispc-1.13.0/CMakeLists.txt.unsupported ispc-1.13.0/CMakeLists.txt
--- ispc-1.13.0/CMakeLists.txt.unsupported	2020-05-31 16:38:08.002098042 +0200
+++ ispc-1.13.0/CMakeLists.txt	2020-05-31 16:50:47.381084594 +0200
@@ -358,7 +358,7 @@ set_target_properties(${PROJECT_NAME} PR
 
 if (UNIX)
     set_target_properties(${PROJECT_NAME} PROPERTIES CXX_EXTENSIONS OFF)
-    target_compile_options(${PROJECT_NAME} PRIVATE -Wno-c99-extensions -Wno-deprecated-register -fno-rtti)
+    target_compile_options(${PROJECT_NAME} PRIVATE -fno-rtti)
     if (ISPC_USE_ASAN)
         target_compile_options(${PROJECT_NAME} PRIVATE -fsanitize=address)
     endif()
